StylableWinFormsControls/StylableWinFormsControls/Controls/StylableTextBox.cs

```diff
@@ -1,111 +1,30 @@
 using System.ComponentModel;
-using System.Reflection;
+using StylableWinFormsControls.Native;
 
 namespace StylableWinFormsControls;
 
 public class StylableTextBox : TextBox
 {
     private System.Windows.Forms.Timer? _mDelayedTextChangedTimer;
-    private string? _hint;
 
     public event EventHandler? DelayedTextChanged;
 
-    private EventHandler? _hintActiveChanged;
-
-    /// <summary>
-    /// will be set when the Text is being set because of Hint logic
-    /// </summary>
-    private bool _hintRefresh;
-
-    /// <summary>
-    /// will be triggered when IsHintActive changes.
-    /// this event will detect when a callback is already registered and will not register again
-    /// </summary>
-    public event EventHandler HintActiveChanged
-    {
-        add
-        {
-            if (_hintActiveChanged?.GetInvocationList().Contains(value) != true)
-            {
-                _hintActiveChanged += value;
-            }
-        }
-        remove => _hintActiveChanged -= value;
-    }
-
     public Color BorderColor { get; set; } = Color.Blue;
 
     /// <summary>
     /// used to set the color of the hint text
     /// </summary>
-    public Color HintForeColor { get; set; } = Color.Gray;
-
-    /// <summary>
-    /// used to set the color of the hint text
-    /// </summary>
-    public Color TextForeColor { get; set; } = Color.Black;
+    public Color PlaceHolderForeColor { get; set; } = Color.Gray;

```

Comment:
I really don't want to be that picky, but I think the difference in upper/lower case of Placeholder vs. PlaceHolder might confuse people searching via autocomplete, wouldn't it?